.github/workflows/release.yml

```diff
@@ -57,3 +95,134 @@ jobs:
           VERSION: ${{ steps.prepare.outputs.tag_name }}
           DOCKER_USERNAME: ${{ secrets.DOCKERHUB }}
           DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_KEY }}
+          
```

Comment:
Commented out new implementation of releasing other binaries of erigon. We are using original method of deploying only erigon to docker, since other components aren't tested.